CHANGELOG.md

```diff
@@ -2,10 +2,349 @@
+  [#3765](https://github.com/mixxxdj/mixxx/pull/3765)
+  [#3849](https://github.com/mixxxdj/mixxx/pull/3849)
+  [#3876](https://github.com/mixxxdj/mixxx/pull/3876)
+  [#3861](https://github.com/mixxxdj/mixxx/pull/3861)
+  [#3923](https://github.com/mixxxdj/mixxx/pull/3923)
+  [#3948](https://github.com/mixxxdj/mixxx/pull/3948)
+  [#3929](https://github.com/mixxxdj/mixxx/pull/3929)
+  [#4007](https://github.com/mixxxdj/mixxx/pull/4007)
+  [#4070](https://github.com/mixxxdj/mixxx/pull/4070)
+  [#4084](https://github.com/mixxxdj/mixxx/pull/4084)
+  [#4098](https://github.com/mixxxdj/mixxx/pull/4098)
+  [#4113](https://github.com/mixxxdj/mixxx/pull/4113)
+  [#4163](https://github.com/mixxxdj/mixxx/pull/4163)
+  [#4166](https://github.com/mixxxdj/mixxx/pull/4166)
+  [#4185](https://github.com/mixxxdj/mixxx/pull/4185)
+  [#4187](https://github.com/mixxxdj/mixxx/pull/4187)
+* Drop Ubuntu Bionic support, require Qt 5.12
+  [#3687](https://github.com/mixxxdj/mixxx/pull/3687)
+  [#3735](https://github.com/mixxxdj/mixxx/pull/3735)
+  [#3736](https://github.com/mixxxdj/mixxx/pull/3736)
+  [#3985](https://github.com/mixxxdj/mixxx/pull/3985)
+* Add NixOS support
+  [#2820](https://github.com/mixxxdj/mixxx/pull/2820)
+  [#2828](https://github.com/mixxxdj/mixxx/pull/2828)
+  [#2836](https://github.com/mixxxdj/mixxx/pull/2836)
+  [#2827](https://github.com/mixxxdj/mixxx/pull/2827)
+  [#2827](https://github.com/mixxxdj/mixxx/pull/2827)
+  [#2828](https://github.com/mixxxdj/mixxx/pull/2828)
+  [#3113](https://github.com/mixxxdj/mixxx/pull/3113)
+  [#3089](https://github.com/mixxxdj/mixxx/pull/3089)
+  [#3545](https://github.com/mixxxdj/mixxx/pull/3545)
 * Add Random Track Control to AutoDJ [#3076](https://github.com/mixxxdj/mixxx/pull/3076)
 * Add support for saving loops as hotcues [#2194](https://github.com/mixxxdj/mixxx/pull/2194) [lp:1367159](https://bugs.launchpad.net/mixxx/+bug/1367159)
+* Make use of inclusive language [#2894](https://github.com/mixxxdj/mixxx/pull/2894)
+* Add Noise effect [#2921](https://github.com/mixxxdj/mixxx/pull/2921)
+* Improve Unittests
+  [#2938](https://github.com/mixxxdj/mixxx/pull/2938)
+  [#2980](https://github.com/mixxxdj/mixxx/pull/2980)
+  [#3006](https://github.com/mixxxdj/mixxx/pull/3006)
+* Logging: Add support for QT_MESSAGE_PATTERN env var [#3204](https://github.com/mixxxdj/mixxx/pull/3204)
+* Colored logging console output and other improvements
+  [#3197](https://github.com/mixxxdj/mixxx/pull/3197)
+  [#3518](https://github.com/mixxxdj/mixxx/pull/3518)
+* Improve command line argument parser
+  [#3640](https://github.com/mixxxdj/mixxx/pull/3640)
+  [#3962](https://github.com/mixxxdj/mixxx/pull/3962)
+  [#4022](https://github.com/mixxxdj/mixxx/pull/4022)
+  [#4036](https://github.com/mixxxdj/mixxx/pull/4036)
+  [#4170](https://github.com/mixxxdj/mixxx/pull/4170)
+  [#4057](https://github.com/mixxxdj/mixxx/pull/4057)
+* Added the Recordbox Hotcue Color Palette [#3746](https://github.com/mixxxdj/mixxx/pull/3746)
```

Comment:
already part of 2.3.0

Reply:
I don't think so, maybe the description was misleading. Improved.

Reply:
Same issue as with the other PR. the PR linked is part of main, but the actual feature / commit has been merged into 2.3 in #3749